libmscore/check.cpp

```diff
@@ -250,6 +251,34 @@ bool Score::sanityCheck(const QString& name)
       return result;
       }
 
+//---------------------------------------------------------
+//   isInvalid
+///    Performes a simple check to check whether the score
+///    contains critical errors which will likely cause
+///    crashes or other serious errors on operating on it.
+//---------------------------------------------------------
+
+bool Score::isInvalid() const
```

Comment:
The common practice is to introduce `isValid()` methods. We can do the same here. Btw, this method looks more like `isEmpty()` than validity check, it only checks number of measures and presence of at least one segment in first measure.
So, I suggest two possible ways: rename it to `isValid()` since checks could be extended in future or `isEmpty()` since it is what this method actually checks now.

Reply:
I wouldn't call it `isValid()` since it is not guaranteed that the score is really valid if this function doesn't consider it invalid. Failing this check doesn't also necessarily mean that the score is empty. As for now, I am not even sure that the proposed patch is the best way to address the described issues. Perhaps I should close this pull request and consider ensuring validity of state of a score or correct handling of such unusual score states by other means.